tests/testthat/test-pmcmc.R

```diff
@@ -0,0 +1,337 @@
+context("pmcmc")
+
+data <- read.csv(system.file("extdata/example.csv", 
```

Comment:
Use `sircovid_file()` instead of `system.file`

Reply:
Put this in each function, rather than using a global

Reply:
done and pushed